athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/data/FieldResolver.java

```diff
@@ -48,12 +48,12 @@
         public Object getFieldValue(Field field, Object value)
         {
             Types.MinorType minorType = Types.getMinorTypeForArrowType(field.getType());
-            if (minorType == Types.MinorType.LIST) {
```

Comment:
just curious why change the order?

Reply:
basically its impossible to get an iterator of this type from a Map so its never going to work. It was supposed to be reversed.